homeassistant/components/nobo_hub/climate.py

```diff
@@ -61,6 +75,17 @@ async def async_setup_entry(
         else nobo.API.OVERRIDE_TYPE_CONSTANT
     )
 
+    # Register hub as device
+    dev_reg = device_registry.async_get(hass)
+    dev_reg.async_get_or_create(
```

Comment:
Please do this from the init module entry setup.

Reply:
Done.